index.html

```diff
@@ -2,12 +2,16 @@
 <head>
 	<title> Column Layout </title>
 	<style type="text/css" media="screen">
+		body {
+			text-align: center;
```

Comment:
This is just so we can see text position perfectly not for any thing else, just unnecessary style to make the page look good.